### ISSUE TITLE
smdb.shore.mbari.org seem not accessible with SCP

### ISSUE THREAD

**Issue by darcosion** [mbari-org/SeafloorMappingDB#265]
Hi, 

I try to use the database and following deployment instruction, I try this command : 
```sh
user@hostname:~/Docs/SMDB$ scp -v smdb.shore.mbari.org:/opt/docker_smdb_vol/SeafloorMapping.db docker_smdb_vol/etc
Executing: program /usr/bin/ssh host smdb.shore.mbari.org, user (unspecified), command sftp
debug1: Reading configuration data /etc/ssh/ssh_config
debug1: /etc/ssh/ssh_config line 19: include /etc/ssh/ssh_config.d/*.conf matched no files
debug1: /etc/ssh/ssh_config line 21: Applying options for *
debug1: Connecting to smdb.shore.mbari.org [134.89.12.160] port 22.
debug1: connect to address 134.89.12.160 port 22: Connection timed out
ssh: connect to host smdb.shore.mbari.org port 22: Connection timed out
scp: Connection closed
```
As you can see, it doesn't end well. 

How can we download the SeafloorMapping.db correctly ? There is some geographical restrictions ? Should I use an American IP ? 

Best regards, 
Darcosion

**Comment by MBARIMike** [comment]
The instructions state:
> This software is placed in the open for the potential benefit of the wider MB-System community. The instructions here are tailored for its development and operation on internal MBARI servers.

Access to the SeafloorMapping.db file is restricted to the internal MBARI network.

**Comment by dwcaress** [comment]
No, software and data are not the same thing. The software that we use for processing and managing seafloor mapping data are open source, including what we develop ourselves (e.g. MB-System: https://www.mbari.org/technology/mb-system/). The SMDB code is openly available through this Github repository, and you are certainly free to adapt it to form a similar website accessing your own data. However, building SMDB yourself is not going to give you access to our network or our private data holdings. Our seafloor mapping datasets are not made publicly available until they are fully processed, interpreted, and published. Published survey data, both raw and processed, are submitted to the national repository MGDS (https://www.marine-geo.org) and can be searched for and downloaded there. Portions of these datasets are also available through NCEI (https://www.ncei.noaa.gov) which obtains multibeam survey swath data from MGDS.

**Comment by darcosion** [comment]
Well, I guess it is useless so, unless I find the way to reconstruct the same database through the MGDS dataset repository...

Thanks for your explanations anyway
